pkg/kubectl/cmd/cmd.go

```diff
@@ -235,6 +236,7 @@ Find more information at https://github.com/kubernetes/kubernetes.`,
 	cmds.AddCommand(NewCmdVersion(f, out))
 	cmds.AddCommand(NewCmdExplain(f, out))
 	cmds.AddCommand(NewCmdConvert(f, out))
+	cmds.SetOutput(os.Stdout)
```

Comment:
@smarterclayton : Currently "kubectl help create" goes to stderr (Please see the bug report). If there's a better way, happy to change the code.